db/cache/charts.go

```diff
@@ -1101,7 +1223,7 @@ func hashrate(time, chainwork ChartUints) (ChartUints, ChartUints) {
 	}
 	t := make(ChartUints, 0, hrLen)
 	y := make(ChartUints, 0, hrLen)
-	var rotator [HashrateAvgLength]uint64
+	rotator := make([]uint64, HashrateAvgLength)
```

Comment:
Why slice vs array?

Reply:
Whoops. I was initially implementing day-binned hashrates in this function as well, but reversed course cause it got messy.